src/pages/LandingPage/components/EventList/components/EventCard/components/CardStatusLabel/styled.ts

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+import { usingTypography } from 'src/hooks/theme';
+
+export const Container = styled.ul`
+  display: flex;
+  justify-content: flex-start;
+  position: relative;
+  color: white;
+`;
+
+export const Label = styled.ul`
+  top: 0;
+  left: 0;
+  position: absolute;
+  //margin: 0;
```

Comment:
Don't forget to remove commented-out code 😛

Reply:
Good catch, will remove it